README.md

```diff
@@ -3,63 +3,116 @@
 </a>
 
 # Block Spam Calls. Powered by Twilio - Python/Flask
-[![Build
-Status](https://travis-ci.org/TwilioDevEd/block-spam-calls-python.svg?branch=master)](https://travis-ci.org/TwilioDevEd/block-spam-calls-python)
+
+![](https://github.com/TwilioDevEd/block-spam-calls-python/workflows/Flask/badge.svg)
 
 > We are currently in the process of updating this sample template. If you are encountering any issues with the sample, please open an issue at [github.com/twilio-labs/code-exchange/issues](https://github.com/twilio-labs/code-exchange/issues) and we'll try to help you.
 
+## About
+
 Learn how to use Twilio add-ons to block spam calls. Learn more about what's needed from this 
 [Twilio guide](https://www.twilio.com/docs/guides/block-spam-calls-and-robocalls-python).
 
-## Local development
+Implementations in other languages:
 
-First you need to have [python](https://www.python-lang.org/), [virtualenv](https://virtualenv.pypa.io) and [pip](https://pip.pypa.io/) installed on your system.
+| .NET | Java | Ruby | PHP | Node |
+| :--- | :--- | :----- | :-- | :--- |
+| [Done](https://github.com/TwilioDevEd/block-spam-calls-csharp) | [Done](https://github.com/TwilioDevEd/block-spam-calls-java)  | [Done](https://github.com/TwilioDevEd/block-spam-calls-ruby)  | [Done](https://github.com/TwilioDevEd/block-spam-calls-php) | [Done](https://github.com/TwilioDevEd/block-spam-calls-node)  |
 
-To run the app locally, clone this repository and `cd` into its directory:
+## Set up
 
-1. First clone this repository and `cd` into its directory:
```

Comment:
fix numbering

Reply:
The enumeration works in this way too. However, I think is a good idea to mention the specific number on each instruction. Thanks!